projects/components/src/quick-search/quick-search.component.html

```diff
@@ -124,7 +130,7 @@ <h5 *ngIf="searchSection.shouldShowText" class="search-result-section-title" dat
     (keydown.enter)="onEnterKey($event)"
     *ngIf="!isPinned; else drawerComponent"
 >
-    <div class="modal-body">
+    <div class="modal-body" [attr.data-ui]="DataUi.quickSearchModal">
```

Comment:
There's no need to prefix the name with quickSearch since it's already contextual to quick search. `modal-body` seems more concise and descriptive